transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -546,6 +554,11 @@ protected void cleanup() {
                 logger.warn("Failed to close the epoll fd.", e);
             }
             try {
+                // Ensure any inflight wakeup writes have been performed prior to closing eventFd.
```

Comment:
It's unclear how this works when eventFdWrite will call eventfd_read:
https://github.com/netty/netty/blob/1fa7a5e697825bdd2f5ad7885b64749ede5c3192/transport-native-epoll/src/main/c/netty_epoll_native.c#L130

Reply:
@ejona86 thank you, I had forgotten about the existence of this other `eventfd_read`. But I think that's essentially dead code given we only ever write a value of `1` at a time into 8 bytes.

I'd propose to replace this check with a comment to make things clearer and just tread it as an error case.

Reply:
Then I think we should explicitly remove it. Norman put it there recently and explicitly. I don't think anything has changed since then that would invalidate it. https://github.com/netty/netty/commit/f6cf681f9004d30ac19c1aa7bb43f628d088e8c9 . I agree that overflowing a 64 bit integer via increments is not going to happen. I don't know if there are other concerns.